src/modules/roc_audio/sample_spec.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2021 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_audio/sample_spec.h
+//! @brief Sample specifications.
+
+#ifndef ROC_AUDIO_SAMPLE_SPEC_H_
+#define ROC_AUDIO_SAMPLE_SPEC_H_
+
+#include "roc_audio/units.h"
+#include "roc_core/time.h"
+#include "roc_packet/units.h"
+
+namespace roc {
+namespace audio {
+
+//! Default sample rate, number of samples per second.
+const size_t DefaultSampleRate = 44100;
+
+//! Default channel mask.
+const packet::channel_mask_t DefaultChannelMask = 0x3;
```

Comment:
It would be nice to keep these defaults in one place (in roc_pipeline/config.h) and not duplicate them here. Is there something stopping us from removing them from here?